modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/frills.dm

```diff
@@ -45,3 +45,13 @@
 	icon_state = "cobraears"
 	icon = 'modular_skyrat/master_files/icons/mob/sprite_accessory/frills.dmi'
 	color_src = USE_MATRIXED_COLORS
+
+/datum/sprite_accessory/frills/neckfrills
+	name = "Neck Frills"
+	icon_state = "neck"
+	icon = 'modular_skyrat/master_files/icons/mob/sprite_accessory/frills.dmi'
+
+/datum/sprite_accessory/frills/neckfrillsfuller
+	name = "Neck Frills (Fuller)"
```

Comment:
I assume the Fuller one is the 1st set of images, going around the whole head?
Wonder if there's a better name for this. None comes to mind for me though